notebooks/WET/WET-013_exposure-time.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    butler = Butler(path_to_aos_butler)
</pre><p>maybe add comment here saying, you will have to change this to run with on_sky images?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Added note.